cmd/cosign/cli/verify/verify.go

```diff
@@ -60,6 +60,8 @@ type VerifyCommand struct {
 	CertGithubWorkflowName       string
 	CertGithubWorkflowRepository string
 	CertGithubWorkflowRef        string
+	CAIntermediates              string
```

Comment:
Unfortunately, we have a lot of duplication in cosign right now. We'll also need to add this to verify-blob, verify-attestation, and verify-blob-attestation. Before you add that, let's wrap up all comments here, and then it should be a straightforward copy-paste.

Reply:
yes, I know - wanted to do it in a follow-up PR to make it easier to review.  Can make an issue not to forget.

Reply:
(TODO - `add this to verify-blob, verify-attestation, and verify-blob-attestation`, after settling on the initial `cosign verify` case)

Reply:
https://github.com/sigstore/cosign/issues/3759